v3/src/components/slider/inspector-panel/slider-settings-panel.tsx

```diff
@@ -46,23 +45,23 @@ console.log(typeof sliderModel.multipleOf)
         <FormControl size="xs">
           <Flex className="palette-row">
             <FormLabel className="form-label">{t("DG.Slider.multiples")}
-              <NumberInput className="slider-input multiples" size="xs"
-                precision={2} step={1} onChange={handleMultiplesOfChange}>
-                  <NumberInputField />
-                  <NumberInputStepper>
-                    <NumberIncrementStepper />
-                    <NumberDecrementStepper />
-                  </NumberInputStepper>
+              <NumberInput className="slider-input multiples" size="xs" defaultValue={sliderModel.multipleOf || 1}
+                  min={0.1} precision={2} step={1} onChange={handleMultiplesOfChange}>
+                <NumberInputField />
+                <NumberInputStepper>
+                  <NumberIncrementStepper />
+                  <NumberDecrementStepper />
+                </NumberInputStepper>
               </NumberInput>
             </FormLabel>
           </Flex>
         </FormControl>
         <FormControl>
           <Flex className="palette-row">
             <FormLabel className="form-label">{t("DG.Slider.maxPerSecond")}
-              <NumberInput className="slider-input animation-rate" size="xs"
-                 precision={2} step={1} onChange={handleAnimationRateChange}>
-                <NumberInputField/>
+              <NumberInput className="slider-input animation-rate" size="xs" min={0.1} precision={2} step={1}
+                  defaultValue={sliderModel.animationRate || 1} onChange={handleAnimationRateChange}>
```

Comment:
These defaults are incorrect. See PT https://www.pivotaltracker.com/story/show/184773626.

Reply:
I'm in the same area in my current slider work so I wouldn't worry about changing it here.